masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+{
+  "properties": {
+    "orderId": {
+      "type": "string",
+      "required": true
+    },
+    "totalReturnAmount": {
+      "type": "integer",
+      "required": true
+    },
+    "sequenceNumber": {
+      "type": "string",
+      "required": true
+    },
+    "status": { "$ref": "#/$defs/status", "required": true },
+    "returnTotals": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "required": true,
+            "enum": ["items", "shipping", "tax"]
+          },
+          "value": {
+            "type": "integer",
+            "required": true
+          }
+        }
+      }
+    },
```

Comment:
The `returnTotals` field is an array with the breakdown of the `totalReturnAmount` field. For the `shipping` item, we can store the proportional value if the store has the flag on.

Still not sure we need a type for `tax`. Should it be included in the `items` value?

Reply:
@filafb I would add discounts

Reply:
I think it falls in the same case as tax. Is it really necessary? I don't see how we would use it down the line. have you thought about any use case?